support-frontend/assets/pages/supporter-plus-landing/twoStepPages/threeTierLanding.tsx

```diff
@@ -259,25 +259,15 @@ const productCatalogDescInclOffers = {
 	},
 };
 
-function getCardData(
-	// TODO - this type could use some refinement
-	productDescription:
-		| (typeof productCatalogDescExclOffers)[keyof typeof productCatalogDescExclOffers]
-		| (typeof productCatalogDescInclOffers)[keyof typeof productCatalogDescInclOffers],
+/**
+ * @deprecated - we should be useing ProductCatalog data types.
+ * TODO - remove this once TsAndCs work of ☝️ types
+ */
+function getPlanCost(
```

Comment:
Next rewrite to T&C's am guessing? Perhaps I should work on removing the `PlanCost`?